ProductsManegements/Inventory.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System.Diagnostics;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
```

Comment:
Please make sure that all the imported namespaces are used and  Clean up unnecessary imports.